### PR TITLE
fix(python/build_tx): make build_tx work again after blockbook protection upgrades

### PR DESCRIPTION
**Pull request by matejcik**
Also improve script type selection.

### CODE REVIEW

**Comment by matejcik** [comment]
folded a subrepo push here just to get the block_fixup check

File: python/tools/build_tx.py [hunk 1]
```diff
@@ -26,6 +26,16 @@
...
 from trezorlib.protobuf import to_dict
 
+SESSION = requests.Session()
+SESSION.headers.update({"User-Agent": "trezorlib"})
+
+BITCOIN_CORE_INPUT_TYPES = {
+    "scripthash": messages.InputScriptType.SPENDADDRESS,
+    "pubkeyhash": messages.InputScriptType.SPENDP2SHWITNESS,
+    "witness_v0_keyhash": messages.InputScriptType.SPENDWITNESS,
+    "witness_v1_taproot": messages.InputScriptType.SPENDP2SHWITNESS,
```
Comment (prusnak) [inline python/tools/build_tx.py:36]: Shouldn't this be `SPENDTAPROOT`?

Comment (prusnak) [reply python/tools/build_tx.py:36]: Also `SPENDADDRESS` and `SPENDP2SHWITNESS` are mixed up?

Comment (matejcik) [reply python/tools/build_tx.py:36]: well :hankey:

**Pull request merged**